contracts/EtherToken.sol

```diff
@@ -14,6 +15,21 @@ contract EtherToken is StandardToken, TokenWithDeposit, Math {
     // disable default function
     function() { revert(); }
 
+    function approveAndCall(address _spender, uint256 _amount, bytes _extraData)
```

Comment:
shouldn't this be part of contract `ERC23`? This seems like quite generic implementation